Packages/Core/Editor/Drawers/AtomDrawer.cs

```diff
@@ -103,7 +103,8 @@ public override void OnGUI(Rect position, SerializedProperty property, GUIConten
                     var buttonWidth = 24;
                     Rect secondButtonRect;
                     Rect firstButtonRect = IMGUIUtils.SnipRectH(restRect, restRect.width - buttonWidth, out secondButtonRect, gutter);
-                    if (GUI.Button(firstButtonRect, "✓"))
+                    if (GUI.Button(firstButtonRect, "✓")
+                        || Event.current.keyCode == KeyCode.Return)
```

Comment:
It's implemented, good catch!

Even if you are in the process of creating multiple atoms at once, pressing enter now only creates the one that is focused. I'm unsure why since they have the same name: perhaps GUI.GetNameOfFocusedControl does a bit more behind the scenes. Anyho, I'm not complaining :)